nexus-repository-apk/pom.xml

```diff
@@ -36,6 +36,11 @@
       <artifactId>nexus-rapture</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
```

Comment:
This should be instead of 
   <dependency>
      <groupId>org.sonatype.goodies</groupId>
      <artifactId>goodies-testsupport</artifactId>
      <scope>test</scope>
    </dependency>
    <dependency>
so please remove  goodies-testsupport

Reply:
Done in commit a557bbf464a5e6d8e4fa9448c8a2251896673035